packages/backend/migration/1657759324648-show-ads.js

```diff
@@ -0,0 +1,11 @@
+export class showAds1657759324648 {
+	name = 'showAds1657759324648'
+
+	async up(queryRunner) {
+		await queryRunner.query(`ALTER TABLE "user_profile" ADD "showAds" boolean NOT NULL DEFAULT false`);
+	}
+
+	async down(queryRunner) {
+			await queryRunner.query(`ALTER TABLE "user_profile" DROP "showAds"`);
+	}
+}
```

Comment:
In store.ts, does `where: account` not store it on the server?

Reply:
It is stored on the server, but not in the `user_profile` table. It will be [stored as a registry item](https://github.com/misskey-dev/misskey/blob/a4b5a0072d427ba2be4170041f98c11f8ea13e44/packages/client/src/pizzax.ts#L118-L122).